src/Microsoft.OpenApi/Models/OpenApiResponse.cs

```diff
@@ -79,7 +78,14 @@ public void SerializeAsV3WithoutReference(IOpenApiWriter writer)
             writer.WriteStartObject();
 
             // description
-            writer.WriteProperty(OpenApiConstants.Description, Description);
+            if (this.Reference == null)
```

Comment:
I'm not sure we want this condition.  The only time this method is called when reference is not null is when it is a top level object in a component map.  In those cases we would still want it to validate.

Reply:
If I used the following OpenAPI Description, it does not generate a description

```
openapi: 3.0.3
info:
  version: 1.0.0
  title: hey
paths:
  /:
    get:
      responses:
        200: 
          $ref: "#/components/responses/success"
components:
  responses:
    success: {}
          
```

Reply:
That is true. It was later that I realized that Reference property is used for both, to signal **reference** to the component, and to **provide** reference to use to point to the component.